src/entities/activity/model/hooks/useAnswerSubmittedToast.ts

```diff
@@ -1,24 +1,9 @@
-import { useEffect } from "react"
-
-import { useLocation } from "react-router-dom"
 import { toast } from "react-toastify"
 
-import { useCustomMediaQuery } from "~/shared/utils"
-
-type Props = {
-  text: string
-}
-
-export const useAnswerSubmittedToast = (props: Props) => {
-  const location = useLocation()
-  const { greaterThanSM } = useCustomMediaQuery()
-
-  useEffect(() => {
-    const isAnswersSubmitted = location.state?.isAnswersSubmitted || false
+export const useAnswerSubmittedToast = () => {
```

Comment:
It looks like you can rename the hook to `useToast` as there is no logic related answer left

Reply:
And it should be moved to the shared folder

Reply:
Already done it in the next PR